inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1168,15 +1168,14 @@ CNNLayerCreator::CNNLayerCreator(const std::shared_ptr<::ngraph::Node>& node): n
         return res;
     });
 
-    REQUIRED_IE_CONVERSION_CREATOR("Broadcast", "Tile");
     REQUIRED_IE_CONVERSION_CREATOR("Interpolate", "Interp");
     REQUIRED_IE_CONVERSION_CREATOR("NormalizeL2", "NormalizeIE");
     REQUIRED_IE_CONVERSION_CREATOR("GroupConvolution", "ConvolutionIE");
     REQUIRED_IE_CONVERSION_CREATOR("GroupConvolutionBackpropData", "DeconvolutionIE");
 
     addSpecificCreator({ "Convolution", "GatherTree", "GRUCell", "GRUSequence", "HardSigmoid",
                       "LRN", "LSTMCell", "LSTMSequence", "NonMaxSuppression", "RNNCell", "RNNSequence", "OneHot",
-                      "Pad", "PriorBoxClustered", "PriorBox", "Proposal", "Selu", "Swish", "Tile"},
+                      "Pad", "PriorBoxClustered", "PriorBox", "Proposal", "Selu", "Swish"},
```

Comment:
Why we need to change legacy API? CPU plug-in doesn't use it. Please revert.

Reply:
Done.